tests/testsuite/cargo_information/path_and_pick_itself/mod.rs

```diff
@@ -0,0 +1,21 @@
+use cargo_test_macro::cargo_test;
+use cargo_test_support::curr_dir;
+
+use super::{cargo_info, init_registry_without_token};
+
+#[cargo_test]
+fn case() {
+    init_registry_without_token();
+    for ver in ["0.1.1+my-package", "0.2.0+my-package"] {
+        cargo_test_support::registry::Package::new("my-package", ver).publish();
+    }
+
+    cargo_info()
+        .arg("cargo-list-test-fixture")
```

Comment:
If we pass `--registry=dummy-registry` here. It will give an unreasonable error:
```console
    Updating `dummy-registry` index
error: could not find `cargo-list-test-fixture` in registry `file:///Volumes/t7/code/cargo-information/target/tmp/cit/t0/registry`
```